gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/exceptions/MaintenanceModeException.java

```diff
@@ -32,7 +32,7 @@ public int getHttpStatusCode() {
 
     @Override
     public String getTechnicalCode() {
-        return "maintenance.node";
```

Comment:
I see no reason not to correct it. But if you think it's "dangerous", we can leave it as is 🤷‍♂️

Reply:
Not risky for me.